File: Python/mahasiswa.py
class Mahasiswa():

    # atribut private
    __nama = "blank"
    __nim = "blank"
    __prodi = "blank"
    __fakultas = "blank"

    # konstruktor
    def __init__(self, nama="blank", nim="blank", prodi="blank", fakultas="blank"):
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    # deklarasi methods
    def setnama(self, nama):
        self.__nama = nama
        
    def getnama(self):
        return self.__nama 

    def setnim(self, nim):
        self.__nim = nim
        
    def getnim(self):
        return self.__nim 

    def setprodi(self, prodi):
        self.__prodi = prodi
        
    def getprodi(self):
        return self.__prodi 

    def setfakultas(self, fakultas):
        self.__fakultas = fakultas
        
    def getfakultas(self):
        return self.__fakultas 

    # untuk menampilkan hasil inputan user
    def keluaran(self):
       
        print ("Nama       : %s" % (self.__nama)) 
        print ("Nim    : %s" % (self.__nim)) 
        print ("Program studi  : %s" % (self.__prodi)) 
        print ("Fakultas   : %s" % (self.__fakultas))
        

File: C++/mahasiswa.cpp
#include<iostream>
#include<string>

using namespace std;
 
class Mahasiswa
{
    private:
        string nama;
        string nim;
        string prodi;
        string fakultas;
        
    public:
		Mahasiswa(){

        }

        // method untuk nama
        void setnama(string nama){
            this->nama = nama;
        }
        string getnama(){
            return this->nama;
        }

        // method untuk nim
        void setnim(string nim){
            this->nim = nim;
        }
        string getnim(){
            return this->nim;
        }

        // method untuk prodi
        void setprodi(int prodi){
            this-> prodi = prodi;
        }
        int getprodi(){
            return this->prodi;
        }

        // method untuk fakultas
        void setfakultas(int i, string fakultas){
            this-> fakultas[i] = fakultas;
        }
        string getfakultas(int i){
            return this->fakultas[i];
        }

        void putdata();

        ~Mahasiswa(){

		}
};